demisto_sdk/commands/common/errors.py

```diff
@@ -139,6 +139,10 @@
         "code": "BA125",
         "related_field": "description",
     },
+    "description_missing_dot_at_the_end": {
```

Comment:
we don't have something similar for this error code already?

Reply:
Moved the error code to be listed under DS - Descriptions.